src/main/java/duchess/Duchess.java

```diff
@@ -0,0 +1,254 @@
+package duchess;
+
+import java.util.Scanner;
+
+/**
+ * The main class used to execute Duchess actions.
+ */
+public class Duchess {
+    /**
+     * Name for the Duchess AI.
+     */
+    public static final String NAME = "Duchess";
+
+    /**
+     * An ArrayList to store text in.
+     */
+    private static TaskList storedTasks = new TaskList();
+
+    /**
+     * Outputs the same string parameter that was passed in.
+     *
+     * @param s - the string to be echoed.
+     */
+    private static void echo(String s) {
+        Ui.duchessPrint(s);
+    }
+
+    /**
+     * Prints all stored text.
+     *
+     * @param s - the string to be stored.
+     */
+    private static void listTasks() {
+        Ui.duchessPrint("Here are the things you said!! ヽ(^o^)丿");
```

Comment:
Perhaps, you could put this print statement under the Ui class? It kind of defeats the point when you still have to type out the string here :(